cmd/goal/application.go

```diff
@@ -995,3 +1023,123 @@ var infoAppCmd = &cobra.Command{
 		}
 	},
 }
+
+var methodAppCmd = &cobra.Command{
+	Use:     "method",
+	Short:   "Invoke a method",
+	Long:    `Invoke a method in an App (stateful contract) with an application call transaction`,
+	Args:    validateNoPosArgsFn,
+	PreRunE: validateNoPosArgsFn,
+	Run: func(cmd *cobra.Command, args []string) {
+		var applicationArgs [][]byte
+
```

Comment:
For method implementation, I don't think it is nice to let `--arg` and `--app-arg` be mixed together, so I additionally added a `methodArgs` string slice to store the inputs, let me know if you go for/against that opinion.